tests.mk

```diff
@@ -47,7 +48,7 @@ ci-dependencies: bats shellcheck xmlstarlet
 setup-deploy-tests:
 ifdef ENABLE_DOKKU_TRACE
 	echo "-----> Enabling tracing"
-	docker exec -ti dokku bash -c 'echo "export DOKKU_TRACE=1" >> /home/dokku/dokkurc'
+	dokku trace:on
```

Comment:
Why this revert? I thought you wanted to test the built artifact?

Reply:
https://github.com/dokku/dokku/pull/3589#issuecomment-510973346

we unit test just the installed version and test deploys against both